.gitignore

```diff
@@ -6,5 +6,9 @@ node_modules
 .env
 .env.*
 !.env.example
+pnpm-lock.yaml
```

Comment:
Why are we getting rid of the pnpm lockfile? Is it a bad idea to allow build environments to be fully in sync?

Reply:
No, it's not. I've had an issue based on conflict by different version of npm, so its mostly habit to not commit it at this point. Would update the PR to include it.